examples/camera-controller/webrtc_manager/WebRTCProviderClient.cpp

```diff
@@ -153,10 +153,10 @@ void WebRTCProviderClient::OnDeviceConnectedFn(void * context, Messaging::Exchan
     }
 }
 
-void WebRTCProviderClient::OnDeviceConnectionFailureFn(void * context, const ScopedNodeId & peerId, CHIP_ERROR err)
+void WebRTCProviderClient::OnDeviceConnectionFailure(void * context, const ScopedNodeId & peerId, CHIP_ERROR err)
```

Comment:
If the connection fails after sending the ProvideOffer, and before the Answer is received, it is probably prudent to dismantle the flow and cleanup.

Reply:
Acked, https://github.com/project-chip/connectedhomeip/issues/38260